static/scss/partials/main.scss

```diff
@@ -31,6 +31,7 @@
   --violet3: #9059ff;
   --violet4: #7542e5;
   --warningRed: #d70022;
+  --successGreen: #00D053; 
```

Comment:
Let's migrate to Nebula's green. (Confirm with Eduardo)

Additionally – let's start a new variable calling it the correct color. `$color-green-50`. 

<img width="792" alt="Screen Shot 2021-07-26 at 12 48 30 PM" src="https://user-images.githubusercontent.com/2692333/127034986-2147befb-2431-4e1a-abce-cf309c24791b.png">


<img width="259" alt="image" src="https://user-images.githubusercontent.com/2692333/127034675-987f0a3a-79bd-4b74-aef0-5c8d9a88c44d.png">

Reply:
Actually – one second thought. Leave this variable (`successGreen`) as is. I'm going to add in all the colors and we can migrate this later. 

If you wanna update the color of the variabel to `#3FE1B0`, go for it.

Reply:
(Issue filed as https://github.com/mozilla/fx-private-relay/issues/961 and assigned to me)